### PR TITLE
ref(ui) Remove now redundant span tags

### PR DESCRIPTION
**Pull request by markstory**
Remove span tags that we no longer need. Tooltip has recently been refactored so that it can work with components in addition to elements. This allows us to remove wrapping dom nodes.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/projectDebugFiles.jsx [hunk 1]
```diff
@@ -134,9 +134,7 @@ class ProjectDebugSymbols extends AsyncComponent {
               {features &&
                 features.map(feature => (
                   <Tooltip key={feature} title={getFeatureTooltip(feature)}>
-                    <span>
-                      <Tag inline>{feature}</Tag>
-                    </span>
+                    <Tag inline>{feature}</Tag>
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/views/projectDebugFiles.jsx]: Was there a reason `Tag`s needed the span?

Comment (markstory) [reply src/sentry/static/sentry/app/views/projectDebugFiles.jsx]: Not that I can see outside of anchoring the tooltip.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/projectDebugFiles.jsx]: I like this phrase, load bearing span. But I don't know what it means lol

**Review by evanpurkhiser — APPROVED** [review]

File: src/sentry/static/sentry/app/views/projectDebugFiles.jsx (hunk 1 above)
Comment (markstory) [reply src/sentry/static/sentry/app/views/projectDebugFiles.jsx]: Loading bearing means something holding the structure up. Another usage would be "Don't knock that wall down it is load bearing". Might be 🇨🇦 vernacular.

Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/views/projectDebugFiles.jsx]: "load bearing" is definitely a phrase ai know, I just wasn't sure in what way a span could be load bearing haha

**Pull request merged**